lib/HttpClient.ts

```diff
@@ -350,6 +350,19 @@ export class HttpClient implements ifm.IHttpClient {
         return info;
     }
 
+    private _isPresigned(requestUrl): boolean {
+        if (this.requestOptions && this.requestOptions.presignedUrlPatterns) {
+            const patterns: RegExp[] = this.requestOptions.presignedUrlPatterns;
+            for (let i: number = 0; i < patterns.length; i += 1) {
```

Comment:
i++?

Reply:
Yeah, I did this because the ESLint style guide prefers +=, but I think that's actually just outside the context of loops. I'll go with ++